packages/react-scripts/package.json

```diff
@@ -58,9 +58,9 @@
     "fs-extra": "^9.0.1",
     "html-webpack-plugin": "4.5.0",
     "identity-obj-proxy": "3.0.0",
-    "jest": "26.6.0",
-    "jest-circus": "26.6.0",
-    "jest-resolve": "26.6.0",
+    "jest": "27.0.0-next.6",
+    "jest-circus": "27.0.0-next.6",
```

Comment:
this is the default in Jest 27, so you can remove the explicit dependency (and https://github.com/facebook/create-react-app/blob/5fc8350c89ff730cdfd067bbd86a90dab393d21b/packages/react-scripts/scripts/utils/createJestConfig.js#L42)

Reply:
Will do that, thank you!